yarn.lock

```diff
@@ -2,30 +2,30 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.8.3", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.8.3":
```

Comment:
Why is the lock being updated when the package.json has not been updated?

Reply:
I had to delete the lock file because I received some error from babel when trying to use yarn start. Not sure why, but deleting the lock file and running yarn install helped